jujugui/static/gui/src/app/components/machine-view/machine-view.js

```diff
@@ -207,7 +214,10 @@ YUI.add('machine-view', function() {
               disabled={props.acl.isReadOnly()}
               type="inline-neutral"
               title="Auto place" />
-            or manually place
+            <p>
+              You can also drag and drop unplaced units to customise your
```

Comment:
The also is because the UI has a button: "[Auto place] You can also drag and drop unplaced units to customise your deployment."